components/MemberDirectory.tsx

```diff
@@ -14,12 +17,23 @@ interface MemberDirectoryProps {
 
 export default function MemberDirectory({ members }: MemberDirectoryProps) {
   const isFiltered =
-    members.filter((mem) => mem.focus?.filter((foc) => foc.active).length > 0)
-      .length > 0;
+    members.filter(
+      (mem) =>
+        mem.focus
+          .concat(mem.industry)
+          .concat(mem.experienceFilter)
+          .concat(mem.regionFilter)
+          ?.filter((foc) => foc.active).length > 0
```

Comment:
Great point and something I mentioned to @taylorkmho as an idea. I thought there maybe should be a toggle between AND / OR for filter selection like you said, but then I was also thinking about the fact that it already **sorts the results by the highest amount of matched filters**, which I thought might enough. 

Then again I'm not too sure though as I could see some scenarios where it appears a bit confusing. The implementation wouldn't be too hard, but I'll leave that decision up to Taylor.

Reply:
Ah my bad Andrew, just now reading again, you're actually suggesting built-in AND/OR functionality which logically makes sense, I'm just having a hard time conceptualizing how we communicate this filtering logic to users via the UI/UX, or whether or not it's important to

Reply:
Actually, that would be even better; I was just saying that the default behavior should OR within a category and AND between categories. An example where this could be helpful, say I'm a recruiter in Oregon looking for someone with 3-5 years of frontend experience seeing a list of 100 people that match any of those search criteria would be misleading when there might only be 5 people that match all of them.

Reply:
Yeah after playing with it a bit, there's something about filtering by multiple categories that _feels_ like more control or communication may be ideal.

I think I'm with Andrew... I wouldn't let this be what blocks you from merging... since someone should be able to deduce how their filters relate to the results below, especially with how the tags "highlight" if they match.